src/APP_PATHS.js

```diff
@@ -12,19 +12,19 @@ const ApiNameBreadcrumb = ({ match }) => {
   return <span>{match.params.apiName.includes("-") ? hyphenatedToTitleCase(match.params.apiName) : camelToTitleCase(match.params.apiName) }</span>
 }
 
-const SearchBreadcrumb = ({match, location}) => {
+const ApiCatalogueBreadcrumb = ({match, location}) => {
   if(location.search){
     return (
       <>
         <span className="govuk-breadcrumbs__list-item">
-          <Link className="govuk-breadcrumbs__link" href={match}>Search</Link>
+          <Link className="govuk-breadcrumbs__link" href={match}>API Catalogue</Link>
         </span>
-        <span className="govuk-breadcrumbs__list-item" aria-current="page">Search for "{decodeURIComponent(location.search.replace("?query=", "").replaceAll("+", " "))}"</span>
+        <span className="govuk-breadcrumbs__list-item" aria-current="page">Search for "{decodeURIComponent(location.search.replace("?search=", "").replaceAll("+", " "))}"</span>
```

Comment:
"?search=" is visible on the page in the breadcrumb here when previously it was not
e.g. Search for "?search=tenure" v.s. Search for "tenure"